common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/IApiResponse.kt

```diff
@@ -32,7 +31,7 @@ abstract class IApiResponse(
     //HTTP status code
     @Transient open val statusCode: Int,
     correlationId: String
-) {
+) : ILoggable {
```

Comment:
Is there a reason for specifying `ILoggable` here, instead of in the subclasses?

Reply:
The higher up the hierarchy we implement `ILoggable`, the less likely it is that future code changes will overlook this. The need to extend `ILoggable` is the only "manual" step in the process, and the most likely place where we can introduce oversights. Thus, by adding it to base classes, I'm hoping it's less likely it'll be overlooked.